src/main/java/com/toptrumps/db/PlayersList.java

```diff
@@ -0,0 +1,26 @@
+package com.toptrumps.db;
+import java.util.*;
+import com.toptrumps.core.player.Player;
+public class PlayersList{
+
+    public PlayersList(){
```

Comment:
We must get rid of this PlayerList class later on

Reply:
Already done that locally :)